tests/api.c

```diff
@@ -1352,31 +1352,24 @@ static int test_wolfSSL_CTX_load_system_CA_certs(void)
 
 #if !defined(NO_FILESYSTEM) && !defined(NO_CERTS) && !defined(NO_WOLFSSL_CLIENT)
     WOLFSSL_CTX* ctx;
+    byte dirValid = 0;
 
     ctx = wolfSSL_CTX_new(wolfSSLv23_client_method());
     if (ctx == NULL) {
         ret = -1;
     }
     if (ret == 0) {
-#ifdef _WIN32
-        if (wolfSSL_CTX_load_system_CA_certs(ctx) != WOLFSSL_NOT_IMPLEMENTED) {
-            ret = -1;
-        }
-#ifdef OPENSSL_EXTRA
-        if (wolfSSL_CTX_set_default_verify_paths(ctx) != WOLFSSL_FAILURE) {
-            ret = -1;
-        }
-#endif /* OPENSSL_EXTRA */
-#else
+    #ifdef _WIN32
+        dirValid = 1;
+    #else
         word32 numDirs;
         const char** caDirs = wolfSSL_get_system_CA_dirs(&numDirs);
 
         if (caDirs == NULL || numDirs == 0) {
-            ret = -1;
+            ret = -2;
```

Comment:
Should we come up with better return codes than `-1` through `-4`?

Reply:
The idea is to correlate the failure with a specific line for debug. It seems like overkill, IMO, to give them named constants for such a small test. But open to suggestions.

Reply:
Added `fprintf` statements to tell the user which function failed.